packages/devtools_app/lib/src/screens/vm_developer/object_inspector_view.dart

```diff
@@ -46,18 +52,59 @@ class _ObjectInspectorViewState extends State<_ObjectInspectorView> {
         Provider.of<VMDeveloperToolsController>(context);
     controller = vmDeveloperToolsController.objectInspectorViewController!
       ..init();
+    tabController = TabController(
+      length: 2,
+      vsync: this,
+      // Disable the animation.
+      animationDuration: const Duration(),
+    );
   }
 
   @override
   Widget build(BuildContext context) {
+    final theme = Theme.of(context);
     return Split(
       axis: Axis.horizontal,
       initialFractions: const [0.20, 0.80],
       children: [
-        ProgramExplorer(
-          controller: controller.programExplorerController,
-          onNodeSelected: _onNodeSelected,
-          title: 'Program Explorer',
+        Column(
+          children: [
+            Container(
+              color: theme.titleSolidBackgroundColor,
+              child: OutlineDecoration(
+                showBottom: false,
+                child: TabBar(
+                  controller: tabController,
+                  tabs: [
+                    Tab(
+                      height: defaultRowHeight,
+                      text: 'Program Explorer',
+                    ),
+                    Tab(
```

Comment:
DevToolsTab provides helpers to respect font size changes. Use that if possible.